packages/trip-form/src/styled.ts

```diff
@@ -8,6 +9,12 @@ import type {
   SubmodeSelectorAndSubComponents
 } from "./types";
 
+const { grey } = colors;
+
+export const baseColor = () => getComputedStyle(document.documentElement).getPropertyValue(
+  "--main-base-color"
+)
+
```

Comment:
Don't we have this function inside core utils already?

Reply:
We do not :(